modules/security-best-practices/pages/rust-canister-development-security-best-practices.adoc

```diff
@@ -10,19 +10,20 @@ The controller of a canister can change / update the canister whenever they like
 
 ==== Recommendation
 
-* To implement an immutable canister smart contract, you cannot be the controller of the canister. Control must be passed to the service nervous system (SNS) or a decentralized governance system.
-* Another option to create an immutable canister smart contract is to remove the canister controller completely. But note that this implies that the canister cannot be upgraded, which may have severe implications in case e.g. a bug would be found.
-* Note that, contrary to some other blockchains, also immutable smart contracts need cycles to run, and they can receive cycles.
+* Consider passing canister control to a decentralized governance system such as the Internet Computer's Service Nervous System (SNS), so that changes to the canister must be proposed and voted upon. If the SNS is used, use an NNS-blessed version running on the SNS subnet, meaning that SNS is under NNS control. The SNS will be available soon.
+* Another option would be to create an immutable canister smart contract by removing the canister controller completely. However, note that this implies that the canister cannot be upgraded, which may have severe implications in case e.g. a bug were found. The option to use a decentralized governance system and thus being able to upgrade smart contracts is a big advantage of the Internet Computer ecosystem.
```

Comment:
nit: " advantage of the Internet Computer ecosystem" compared to what?
(I assume here we mean "compared to other blockchains", but maybe we don't want to be that straightforward, so leaving it might be the better option?)

Reply:
I like "compared to other blockchains" and added this.